lib/manager/actions/status.js

```diff
@@ -134,6 +136,51 @@ export async function fetchAppInfo () {
   }
 }
 
+function getMergeFeedModalContent (result: MergeFeedsResult): ModalContent {
+  const details = []
+  // Do nothing or show merged feed modal? Feed version is be created
+  details.push('Remapped ID count: ' + result.remappedReferences)
+  if (Object.keys(result.remappedIds).length > 0) {
+    const remappedIdStrings = []
+    for (let key in result.remappedIds) {
+      // Modify key to remove feed name.
+      const split = key.split(':')
+      const tableAndId = split.splice(1, 1)
+      remappedIdStrings.push(`${tableAndId.join(':')} -> ${result.remappedIds[key]}`)
+    }
+    details.push('Remapped IDs: ' + remappedIdStrings.join(', '))
+  }
+  if (result.skippedIds.length > 0) {
+    const skippedRecordsForTables = {}
+    result.skippedIds.forEach(id => {
+      const table = id.split(':')[0]
+      if (skippedRecordsForTables[table]) {
+        skippedRecordsForTables[table] = skippedRecordsForTables[table] + 1
+      } else {
+        skippedRecordsForTables[table] = 1
+      }
+    })
+    const skippedRecordsStrings = []
+    for (let key in skippedRecordsForTables) {
+      skippedRecordsStrings.push(`${key} - ${skippedRecordsForTables[key]}`)
+    }
+    details.push('Skipped records: ' + skippedRecordsStrings.join(', '))
+  }
+  if (result.idConflicts.length > 0) {
+    // const conflicts = result.idConflicts
+    details.push('ID conflicts: ' + result.idConflicts.join(', '))
+  }
+  return {
+    title: result.failed
+      ? 'Warning: Errors encountered during feed merge!'
+      : 'Feed merge was successful!',
+    body: result.failed
+      ? `Merge failed with ${result.errorCount} errors. ${result.failureReasons.join(', ')}`
+      : `Merge was completed successfully. A new version will be processed/validated containing the resulting feed.`,
+    detail: details.join('\n')
+  }
+}
+
 /* eslint-disable complexity */
```

Comment:
Can you remove `/* eslint-disable complexity */` now?

Reply:
I'm not sure if this is needed anywhere anymore. When I remove these `yarn lint` runs fine. @evansiroky can you weigh in on whether these are still needed?